CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ A breaking change will get clearly marked in this log.
 
 ## Unreleased
 
+## [v3.3.0](https://github.com/stellar/js-stellar-sdk/compare/v3.2.0...v3.3.0)
+
+### Deprecated ⚠️
+
+- Horizon 0.25.0 will change the data type for multiple attributes from `Int64` to
+  `string`. When the JSON payload includes an `Int64`, there are
+  scenarios where there might incorrect data since JavaScript doesn't support
+  `Int64` values. You can read more about it in [#1363](https:/.
```

Comment:
URL to the issue looks broken, is it right?

Reply:
fixed!